### PR TITLE
Add Pitchforks as a craftable blacksmithing item.

### PR DESCRIPTION
**Pull request by ShiftyRail**
What it says on the tin.
I would have added torches too, but I can't find the .dmi for it.

:cl:
- rscadd: You can now craft pitchforks with blacksmithing.

### CODE REVIEW

File: code/WorkInProgress/MadmanMartian/blacksmithing/misc_components.dm [hunk 1]
```diff
@@ -44,6 +44,12 @@
 	desc = "unlike the shark, this one lacks bite."
 	result = /obj/item/weapon/hammer
 
+/obj/item/item_head/pitchfork_head
+	name = "pitchfork head"
+	icon_state = "pitchfork_head"
+	desc = "the revolution is not going to start itself."
```
Comment (Kurfursten) [inline code/WorkInProgress/MadmanMartian/blacksmithing/misc_components.dm]: capitalize?

File: code/game/objects/items/weapons/weaponry.dm [hunk 2]
```diff
@@ -460,3 +460,12 @@ obj/item/weapon/banhammer/admin
 	inhand_states = list("left_hand" = 'icons/mob/in-hand/left/hammer_left.dmi', "right_hand" = 'icons/mob/in-hand/right/hammer_right.dmi')
 	force = 8
 	hitsound = 'sound/weapons/toolbox.ogg'
+
+/obj/item/weapon/pitchfork
+	name = "pitchfork"
+	desc = "down with the current state of things!"
```
Comment (Kurfursten) [inline code/game/objects/items/weapons/weaponry.dm]: capitalize?

File: code/game/objects/items/weapons/weaponry.dm [hunk 3]
```diff
@@ -460,3 +460,12 @@ obj/item/weapon/banhammer/admin
...
+	icon = 'icons/obj/weapons.dmi'
+	icon_state = "pitchspoon"
+	inhand_states = list("left_hand" = 'icons/mob/in-hand/left/swords_axes.dmi', "right_hand" = 'icons/mob/in-hand/right/swords_axes.dmi')
+	force = 8
```
Comment (Kurfursten) [inline code/game/objects/items/weapons/weaponry.dm:470]: this really probably needs a sharpness value too...

File: code/WorkInProgress/MadmanMartian/blacksmithing/misc_components.dm (hunk 1 above)
Comment (ShiftyRail) [reply code/WorkInProgress/MadmanMartian/blacksmithing/misc_components.dm]: the rest of the items aren't, but sure.

**Pull request merged**